### PR TITLE
adding result div RELATES #18

### PR DESCRIPTION
**Pull request by sallamTanna**

### CODE REVIEW

File: public/js/mainEditor.js [hunk 1]
```diff
@@ -23,6 +23,16 @@ const navbarPosition = select(".navbarPosition");
 const navbarPreview = select(".cp-navbar-container");
```
Comment (NoureldeanSaed) [inline public/js/mainEditor.js]: Unused variable

File: public/js/mainEditor.js [hunk 2]
```diff
@@ -23,6 +23,16 @@ const navbarPosition = select(".navbarPosition");
...
 const paddingInputForNavbar = select('.paddingInputForNavbar')
 const generateCode = select('.generateCodeButton');
+const backToEditing = select('.backToEditing');
+const navEditContainer = select('.navEditContainer');
+const navPreview = select('.navPreview');
+const resultContainer = select('.resultContainer');
+const finalCssCode = select('.finalCssCode');
```
Comment (NoureldeanSaed) [inline public/js/mainEditor.js]: Unused variable

File: public/js/mainEditor.js [hunk 3]
```diff
@@ -23,6 +23,16 @@ const navbarPosition = select(".navbarPosition");
...
+const navbarHeight = select('.navbarHeight');
```
Comment (NoureldeanSaed) [inline public/js/mainEditor.js]: Unused variable

File: public/js/mainEditor.js [hunk 4]
```diff
@@ -23,6 +23,16 @@ const navbarPosition = select(".navbarPosition");
...
+const GenerateCodeDiv = select('.GenerateCodeDiv');
+
+
+
 const cssCode = select('.cssCode');
```
Comment (NoureldeanSaed) [inline public/js/mainEditor.js]: Unused variable

File: public/js/mainEditor.js (hunk 2 above)
Comment (sallamTanna) [reply public/js/mainEditor.js]: We will use them when we want to display the css and html code, should I remove them now???
@NoureldeanSaed

File: public/js/mainEditor.js (hunk 1 above)
Comment (sallamTanna) [reply public/js/mainEditor.js]: Done

**Pull request merged**